README-de.adoc

```diff
@@ -0,0 +1,457 @@
+Zusätliche Finanzierung wird von unseren https://asciidoctor.org/supporters[Community Backers] bereit gestellt.
+
+Sie können dieses Projekt unterstützen indem sie ein Sponsor auf https://opencollective.com/asciidoctor[OpenCollective] werden.
+
+Das Big Picture
+
+Asciidoctor liest in Klartext geschriebene Inhalte, wie in dem Bild unten links gezeigt, und übersetzt es in HTML5, wie im rechten Bild gerendert und gezeigt.
+Asciidoctor wendet ein Standard-Stylesheet an dem HTML5-Dokument an um eine angenehme Out-of-the-Box Erfahrung zu liefern.
+
+image::{image-uri-screenshot}[Preview of AsciiDoc source and corresponding rendered HTML]
+
+== AsciiDoc-Verarbeitung
+
+Asciidoctor liest und analysiert in Asciidoctor-Syntax geschrieben Klartext, um ihn dann in einen Syntaxbaum zu füttern, mit einem Set aus eingebauten Konvertern, um HTML5, Docbook 5 und Gebrauchsanleitungseiten zu produzieren.
+Sie haben die Option ihren eigenen Konverter zu nutzen oder {uri-tilt}[Tilt]-unterstützende Vorlagen zu laden um den generierten Output zu anzupassen oder zusätzliche Formate zu produzieren.
+
+Asciidoctor ist ein Drop-In-Ersatz für den ursprünglichen AsciiDoc Python Prozessor (`asciidoc.py`).
+Das Asciidoctor Testpaket hat {uri-tests}[> 2,000 Tests] um Kompabilität mit der Asciidoc-Syntax zu gewährleisten.
+
+Zusätzlich zu der klassischen AsciiDoc-Syntax erkennt Asciidoctor zusätzliche Markup- und Formatierungsoptionen, wie die font-basierten Symbole (z.B. `+icon:fire[]+`) und UI-Elemente (z.B. `+button:[Save]+`).
+Asciidoctor bietet auch ein modernes, responsive Theme, basierend auf {uri-foundation}[Foundation] um den HTML5-Output zu stylisieren.
+
+== Wohin Ruby auch geht, Asciidoctor folgt
+
+Sie können Asciidoctor auf der JVM laufen lassen mit JRuby.
+Um die Asciidoctor API direkt aus Java und andere JVM-Sprachen zu aufzurufen, benutzen Sie {uri-asciidoctorj}[AsciidoctorJ].
+Es stehen Ihnen Plugins zur Verfügung, basierend auf AsciidoctorJ, die den Asciidoctorprozessor in Apache Maven, Gradle oder Javadoc Builds integrieren.
+
+Asciidoctor läuft auch in JavaScript.
+{uri-opal}[Opal] wird benutzt um die Ruby Quelle in JavaScript zu transkompilieren um {uri-asciidoctorjs}[Asciidoctor.js] zu produzieren.
+Asciidoctor.js ist eine voll-funktionale Version von Asciidoctor die in jeder JavaScript Umgebung funktioniert, wie einem Webbrowser oder Node.js.
+Er wird benutzt um die AsciiDoc Vorschauerweiterungen für Chrome, Atom, Brackets und andere web-basierte Tooling zu nutzen.
+
+== Anforderungen
+
+Asciidoctor läuft auf Linux, MacOS und Windows und erforderte eine der folgenden Implementationen von {uri-ruby}[Ruby]:
+
+* Ruby (MRI/CRuby 1.8.7 - 2.5)
+* JRuby (1.7 in Ruby 1.8 und 1.9 modes, 9000)
+* Rubinius 2.2.x
+* Opal (JavaScript)
+
+[VORSICHT]
```

Comment:
Admonition labels should not be translated. Can you change VORSICHT back to CAUTION?